scripts/flux.js

```diff
@@ -0,0 +1,50 @@
+export default class Flux {
```

Comment:
Not exactly, if we map the Flux concept here, the dispatcher role is actually played by the event emitter; Flux is the whole container for the stores and the dispatcher. But I totally get what you mean… Actually if we don't want too much to stick to the Flux lexical scope, *Controller* would work pretty well too.